docs/marketplace-required-csv-annotations.md

```diff
@@ -0,0 +1,85 @@
+# Marketplace Required CSV Annotations
+
+An operator's CSV must contain the following annotations for it to be displayed properly within the Marketplace UI:
+
+```yaml
+metadata:
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: The operator's certification. This field can be omitted until a process to get this value is defined at a later date.
```

Comment:
Please see https://github.com/operator-framework/operator-marketplace/pull/68#discussion_r246565550. This defaults to `false` for now until the process is defined.

Reply:
Mentioned that the field should be set to false until the process is defined.